router/handle.go

```diff
@@ -230,6 +238,22 @@ func (rt *Handle) pickup(ctx context.Context, partition string, workers []*worke
 		lastJob = job
 		slot, err := rt.findWorkerSlot(ctx, workers, job, blockedOrderKeys)
 		if err == nil {
+			traceParent := gjson.GetBytes(job.Parameters, "traceparent").String()
+			if traceParent != "" {
+				if _, ok := traces[traceParent]; !ok {
```

Comment:
Do we need special handling if traceParent is empty? It is not something we do in the processor.

Reply:
If traceParent is empty for some reason, we can't probably trace them back.

Reply:
> It is not something we do in the processor.

Are u referring to in here? https://github.com/rudderlabs/rudder-server/blob/d4eca46bfe04d6335b34721496469d4836d7ae98/processor/processor.go#L1562

Reply:
It got missed. Added now.